tests/Guesser/FilterTypeGuesserTest.php

```diff
@@ -40,7 +41,11 @@ class FilterTypeGuesserTest extends TestCase
     protected function setUp(): void
     {
         $this->guesser = new FilterTypeGuesser();
-        $this->modelManager = $this->createStub(ModelManager::class);
+        $this->modelManager = $this->getMockBuilder(ModelManagerInterface::class)
```

Comment:
Maybe we should create a DoctrineORMAdminBundle/ModelManagerInterface

Reply:
yeah It looks like it... we are using methods not defined on `ModelManagerInterface`

Reply:
Shall I create an issue about it?

Reply:
Or maybe move this method somewhere else ?

Reply:
> Shall I create an issue about it?

Yes

Reply:
See https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/1296